examples/simple_example/README.md

```diff
@@ -0,0 +1,33 @@
+# simple_example
+
+## Introduction
+This example is generated from the Surround CLI without the web runner support.
```

Comment:
What about adding instructions for how to use poetry with doit? Do we need to have doit and poetry?

Reply:
Would be good if we can just use one tool.

Reply:
The only reason is that `doit` is only available in the CLI dependencies. If CLI is installed then we can just use the wrapped commands to run things.
But I can add instructions for `doit` by manually installing it if needed.